.github/workflows/raindrops-ci.yml

```diff
@@ -12,6 +12,7 @@ defaults:
 
 jobs:
   build:
+    if: ${{ !github.event.pull_request.draft }}
```

Comment:
Awesome!

Reply:
Another thing I'd like to do is make more of the build steps parallel, like lint/format/build can all just run at the same time IMO

Reply:
I get antsy waiting around lol